src/components/core.ts

```diff
@@ -194,6 +194,7 @@ export default class Core {
     this.config.onReady = this.config.onReady || ((): void => {});
     // eslint-disable-next-line @typescript-eslint/no-empty-function
     this.config.onChange = this.config.onChange || ((): void => {});
+    this.config.onChangeDebounceTime = this.config.onChangeDebounceTime || undefined;
```

Comment:
If `onChangeDebounceTime === 0`, result of the expression id undefined, so there is no way to set 0 as value

Reply:
Thank you @gohabereg, the `undefined` here is handled on the changes I have made in the modificationsObserver.ts file